aexpect/rss_client.py

```diff
@@ -0,0 +1,543 @@
+#!/usr/bin/python
```

Comment:
The license here as well...

Reply:
Could it be that you have some cache which didn't update correctly? This license is available as well. I just double checked at the "Files" from the menu in this PR.

Reply:
We have author here, so I added this, we only don't have sources for authors for the `remote.py` utility.